CHANGE_LOG.md

```diff
@@ -17,7 +19,6 @@ All notable changes to this project will be documented in this file.
 ### Fixed
 - Fix system message dynamic themeing capability
 - Fix systems messages are not being part of markdown rendering for active links
-- Fix `suggestedActions` with `to` property not rendering by passing `userID` to `Composer`
```

Comment:
We might want to leave this in, so it doesn't confuse the reader, and they have a reference to what was reverted when